docs/integrations/sources/google-ads.md

```diff
@@ -4,44 +4,42 @@
 If you don't already have a developer token from Google Ads, make sure you follow the [instructions](#how-to-apply-for-the-developer-token) so your request doesn't get denied.
 {% endhint %}
 
-## Sync overview
+## Features
 
-This source can sync data for the [Google Ads](https://developers.google.com/google-ads/api/fields/v8/overview).
+| Feature | Supported? |
+| :--- | :--- |
+| Full Refresh Sync | Yes |
+| Incremental Sync | Yes |
+| Replicate Incremental Deletes | No |
+| SSL connection | Yes |
 
-### Output schema
+## Supported Tables
 
-This source is capable of syncing the following streams:
+This source is capable of syncing the following tables and their data:
 
-*Main streams*:
+#### Main Tables
 * [accounts](https://developers.google.com/google-ads/api/fields/v8/customer)
 * [ad_group_ads](https://developers.google.com/google-ads/api/fields/v8/ad_group_ad)
 * [ad_groups](https://developers.google.com/google-ads/api/fields/v8/ad_group)
 * [campaigns](https://developers.google.com/google-ads/api/fields/v8/campaign)
 
-*Report streams*
+#### Report Tables
 * [account_performance_report](https://developers.google.com/google-ads/api/docs/migration/mapping#account_performance)
 * [ad_group_ad_report](https://developers.google.com/google-ads/api/docs/migration/mapping#ad_performance)
 * [display_keyword_report](https://developers.google.com/google-ads/api/docs/migration/mapping#display_keyword_performance)
 * [display_topics_report](https://developers.google.com/google-ads/api/docs/migration/mapping#display_topics_performance)
 * [shopping_performance_report](https://developers.google.com/google-ads/api/docs/migration/mapping#shopping_performance)
 
-### Features
+## Getting Started (Airbyte-Cloud)
 
-| Feature | Supported? |
-| :--- | :--- |
-| Full Refresh Sync | Yes |
-| Incremental Sync | Yes |
-| Replicate Incremental Deletes | No |
-| SSL connection | Yes |
-| Namespaces | No |
-
-### Performance considerations
-
-This source is constrained by whatever API limits are set for the Google Ads that is used. You can read more about those limits in the [google developer docs](https://developers.google.com/google-ads/api/docs/best-practices/quotas).
+1. Get the customer ID for your account. Learn how to do that [here](https://support.google.com/google-ads/answer/1704344)
```

Comment:
there should also be a step saying to select `Sign in via Google` and clicking the authorize button

Reply:
Ah. yes! Done.